src/frontend/reducer/notesReducer.js

```diff
@@ -3,6 +3,8 @@ const NotesInitialState = {
   notes: [],
```

Comment:
I see this same file twice. Is this a mistake?

Reply:
I think it is, you can look into this.

Reply:
two files because one file with the name NotesReducer.js and the other with notesReducer.js ... when I changes case in filename or folder name it came twice on Github , will remove not needed file when I complete everything